packages/ui/src/models/camel-entities/flows/routes.ts

```diff
@@ -86,3 +88,62 @@ export class CamelRoute implements BaseVisualCamelEntity {
     return node;
   }
 }
+
+export class KameletBinding implements BaseVisualCamelEntity {
+  readonly id = uuidv4();
+  readonly type = EntityType.KameletBinding;
+
+  constructor(public route: Partial<KameletBindingModel> = {}) {}
+
+  /** Internal API methods */
+  getId(): string {
+    return this.route.id ?? '';
```

Comment:
I'm not sure about this id - it's not a field in KameletBindings, but the BaseVisualCamelEntity defines "getId()" - should I remove it from there?

Reply:
It's a good question, the idea behind it is to provide an `id` to distinguish each entity at the Visual level, maybe it won't be needed.

That being said, at this point, I'm not 100% sure that it will be used in the future, if you don't mind, can we implement said method by simply returning an empty string in the meantime for `KameletBindings`?

Reply:
sure - updated